azure-pipelines/signing.yml

```diff
@@ -473,6 +473,42 @@ jobs:
         PathtoPublish: '$(Build.ArtifactStagingDirectory)\drop'
         ArtifactName: 'Drop'
         publishLocation: 'Container'
+    - task: CopyFiles@2
+      displayName: 'Copy Packaging Solution to Staging Area'
+      inputs:
+        SourceFolder: '$(Build.SourcesDirectory)/azure-pipelines/vs-insertion'
+        TargetFolder: '$(Build.ArtifactStagingDirectory)/vs-insertion/staging'
+        Contents: 'vcpkg.nuspec'
+        CleanTargetFolder: true
+    - task: CopyFiles@2
+      displayName: 'Copy vcpkg drop to Staging Area'
+      inputs:
+        SourceFolder: '$(Build.ArtifactStagingDirectory)/drop'
+        TargetFolder: '$(Build.ArtifactStagingDirectory)/vs-insertion/staging'
+        Contents: |
+          vcpkg-standalone-bundle.tar.gz
+          vcpkg.exe
+    - task: PowerShell@2
+      displayName: 'Extract vcpkg standalone bundle'
+      inputs:
+        targetType: 'inline'
+        script: |
+          tar.exe -C $(Build.ArtifactStagingDirectory)/vs-insertion/staging/ -xf $(Build.ArtifactStagingDirectory)/vs-insertion/staging/vcpkg-standalone-bundle.tar.gz

```

Comment:
```suggestion
          tar.exe -C $(Build.ArtifactStagingDirectory)/vs-insertion/staging/ -xf $(Build.ArtifactStagingDirectory)/drop/vcpkg-standalone-bundle.tar.gz
```

Reply:
I think this name "drop" in the root of artifactstagingdirectory may already be being used?